hack/e2e-kind-cluster-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -xe
+
+CNI_VERSION=${CNI_VERSION:-0.4.0}
+OCI_BIN=${OCI_BIN:-docker}
+IMG_REGISTRY=${IMAGE_REGISTRY:-localhost:5000/maiqueb}
```

Comment:
note that sometimes localhost doesn't work and 127.0.0.1 does
(not sure why, saw locally on fedora 35+, so if you use 127.0.0.1 it might be easier to run locally as well)

Reply:
I have never seen that happen. 

I am - afaiu - only following up on the empiric best practices I have seen, and documentation of the container registry: https://docs.docker.com/registry/deploying/#copy-an-image-from-docker-hub-to-your-registry